cargo-dist/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "cargo-dist"
 description = "Shippable application packaging for Rust"
-version = "0.1.0"
+version = "0.2.0-prerelease.1"
```

Comment:
cargo-dist-schema and cargo-dist are always versioned in lockstep (this is why i like using `cargo release` to automate that).

Reply:
(fixing this would be #180)